indexer/src/test/java/au/org/aodn/esindexer/configuration/GeoNetworkSearchTestConfig.java

```diff
@@ -126,4 +125,24 @@ public GeoNetworkServiceImpl createGeoNetworkServiceImpl(
 
         return impl;
     }
+    /**
+     * Hardcode the result here for testing, please add more if you need to return different result
+     * @return
+     */
+    @Bean
+    public UrlUtils createUrlUtils() {
+        UrlUtils urlUtils = Mockito.mock(UrlUtils.class);
+
+        final Map<String, Boolean> status = new HashMap<>();
+        status.put("https://catalogue-imos.aodn.org.au/geonetwork/images/logos/dbee258b-8730-4072-96d4-2818a69a4afd.png", Boolean.TRUE);
```

Comment:
The links are coming from data GN4, during harvesting, so it is not something hardcode. If catalogue gone then, we will not see any link pointing there. (Well assume raw data from Harvester is correct)

In the code GeoNetworkServiceImpl, there is a filter check 

```
                            .stream()
                            .map(p -> (p instanceof String) ? (String) p : null)
                            .filter(Objects::nonNull)
                            .filter(i -> urlUtils.checkUrlExists(i))
```

to include the first reachable url, so in case the link really broken, it will not be included.